packages/labs/gen-wrapper-vue/test-output/src/tests/Container.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+import ElementA from '@lit-internal/test-element-a-vue/ElementA.js';
+
+const props = defineProps<{foo?: string}>();
+</script>
+
+<template>
+  <header>Container</header>
+  <ElementA :foo="foo" />
```

Comment:
we'll want to do this for all the generators, but we eventually want failing type-checks to make sure we didn't do something overly permissive with types.

Reply:
Added TODO.